jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -610,8 +609,6 @@ public JibContainerBuilder applyContainerBuildPlan(ContainerBuildPlan buildPlan)
       throws InvalidImageReferenceException {
     containerBuildPlanBuilder
         .setBaseImage(buildPlan.getBaseImage())
-        .setArchitectureHint(buildPlan.getArchitectureHint())
-        .setOsHint(buildPlan.getOsHint())
```

Comment:
I think this should call `setPlatforms(buildPlan.getPlatforms())`.

Reply:
I had done it here  https://github.com/GoogleContainerTools/jib/blob/0ab161a7787210925a2bb3af4d43884bfdeadcdc/jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java